.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules/
+build/
+/docs/
+.coverage/
+npm-debug.log
+.nyc_output
+.vscode
+.idea
+.DS_Store
+package-lock.json
```

Comment:
We want package-lock.json to be checked in. I don't think it should be ignored then?

Reply:
Correct. Reverted.